jupyter_sphinx/execute.py

```diff
@@ -195,6 +229,12 @@ def cell_output_to_nodes(cell, data_priority, dir):
                     rawsource=data,
                     language='ipython',
                 ))
+            elif 'javascipt' in mime_type or 'json' in mime_type:
```

Comment:
`javascipt` -> `javascript`

only took me about half an hour to find this bug :joy:

Reply:
and it should be 
```python
                if 'json' in mime_type:
                    data = json.dumps(data)
                to_add.append(docutils.nodes.raw(
                    text='<script type="{mime_type}">{data}</script>'
                         .format(mime_type=mime_type, data=data),
```

Reply:
Let's be more strict with mime type checking and verify for exact equality whenever possible.
So let's only treat `application/javascript` as javascript.

Also, is embedding arbitrary json in a script the right thing to do? It certainly won't work for a plotly figure, there one needs something like [this](https://gitlab.kwant-project.org/solidstate/lectures/blob/master/extra_markdown.tpl#L3-26). It also will unlikely produce an expected result when somebody just outputs JSON without any specialized meaning.